### ISSUE TITLE
Allow color prop to be empty?

### ISSUE THREAD

**Issue by yangg** [casesandberg/react-color#311]
Hi, color prop has
```
ColorPicker.defaultProps = {
    color: {
      h: 250,
      s: 0.50,
      l: 0.20,
      a: 1,
    },
  }
```
it it possible to allow this prop to be empty?
Like allowEmpty option in this https://bgrins.github.io/spectrum/
Thanks!

**Comment by casesandberg** [comment]
Are you looking for some sort of UI element that allows the user to select `empty`? You could always just pass down a color with 0 alpha and that would be considered empty.

**Comment by rgnevashev** [comment]
+1 https://bgrins.github.io/spectrum/#toc25
It would be good having option "clear color"

**Comment by yangg** [comment]
@casesandberg  Allow the user to select empty, and allow the color prop to be empty(currently has default value).

I've tried to pass `rgba(0, 0, 0, 0)`, but the result is I have to drag the 'alpha slider' to let my selected color take into effect(visible), this seems not user friendly.

**Comment by casesandberg** [comment]
This is now available in `react-color@2.12.0`

You can use it by passing a swatch down as 'transparent' instead of a hex color!

You can also initialize a color picker by passing in `{ hex: 'transparent' }`
